WeaveApp/webpack.prod.js

```diff
@@ -0,0 +1,23 @@
+const merge = require('webpack-merge');
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
+const Visualizer = require('webpack-visualizer-plugin');
+const common = require('./webpack.common.js');
+const webpack = require('webpack');
+
+module.exports = merge(common, {
+    devtool: 'source-map',
+    plugins: [
+        new UglifyJSPlugin({
+            sourceMap: true,
+        }),
+        new webpack.DefinePlugin({
+            'process.env': {
+                'NODE_ENV': JSON.stringify('production'),
+            },
+        }),
+        new webpack.HashedModuleIdsPlugin(),
+        new Visualizer({
```

Comment:
This plugin creates a static file, called `statistics.html`, that creates a visualization of our application and all its sub dependencies. This is especially useful for identifying dependencies that are particularly heavy, and may be targets for replacement by lighter alternatives.